testing/content/files/en-us/web/fixable_flaws/index.html

```diff
@@ -0,0 +1,19 @@
+---
+title: 'Fixable flaws'
+slug: Web/Fixable-Flaws
+summary: Contains flaws that can be fixed.
+---
+
+<p>Some macros</p>
+<ul>
+  <li>{{CSSxRef('dumber')}}</li>
+  <li>{{htmlattrxref("href", "anchor")}}</li>
```

Comment:
This is a good choice, but it requires https://github.com/mdn/yari/pull/822 to work properly. Without https://github.com/mdn/yari/pull/822, the fix would be:
```
{{htmlattrxref("/en-US/docs/Web/HTML/Element/a", "anchor")}}
```
which isn't correct. Once https://github.com/mdn/yari/pull/822 lands, the fix will be:
```
{{htmlattrxref("href", "a")}}
```
which is correct.